internal/params/config.go

```diff
@@ -19,20 +20,33 @@ var (
 	// TestnetChainConfig contains the chain parameters to run a node on the harmony test network.
 	TestnetChainConfig = &ChainConfig{
 		ChainID:        big.NewInt(2),
-		CrossLinkEpoch: big.NewInt(1),
+		CrossTxEpoch:   big.NewInt(1),
+		CrossLinkEpoch: big.NewInt(2),
 		EIP155Epoch:    big.NewInt(0),
 		S3Epoch:        big.NewInt(0),
 	}
 
 	// AllProtocolChanges ...
 	// This configuration is intentionally not using keyed fields to force anyone
 	// adding flags to the config to also have to set these fields.
-	AllProtocolChanges = &ChainConfig{big.NewInt(100), big.NewInt(0), big.NewInt(0), big.NewInt(0)}
+	AllProtocolChanges = &ChainConfig{
+		big.NewInt(100), // ChainID
+		big.NewInt(0),   // CrossTxEpoch
+		big.NewInt(0),   // CrossLinkEpoch
+		big.NewInt(0),   // EIP155Epoch
+		big.NewInt(0),   // S3Epoch
+	}
 
 	// TestChainConfig ...
 	// This configuration is intentionally not using keyed fields to force anyone
 	// adding flags to the config to also have to set these fields.
-	TestChainConfig = &ChainConfig{big.NewInt(99), big.NewInt(0), big.NewInt(0), big.NewInt(0)}
+	TestChainConfig = &ChainConfig{
+		big.NewInt(99), // ChainID
```

Comment:
declaring as constants would look better, like in mainnet/testnet/localnet.go, etc.

Reply:
#1551